pkg/syncer/cnsoperator/manager/init.go

```diff
@@ -382,7 +383,7 @@ func watcher(ctx context.Context, cnsOperator *cnsOperator) error {
 // with the latest configInfo.
 func reloadConfiguration(ctx context.Context, cnsOperator *cnsOperator) error {
 	log := logger.GetLogger(ctx)
-	cfg, err := common.GetConfig(ctx)
+	cfg, err := util.GetConfig(ctx)
```

Comment:
CNS Operator is for is mainly for WCP. Can we avoid making changes to this file?

Reply:
Removed this change since this code is related to WCP.